website/siteConfig.js

```diff
@@ -10,8 +10,8 @@
 
 const siteConfig = {
   title: "Hack" /* title for your website */,
-  /* tagline:  "foo" , */
-  disableTitleTagline: true,
+  tagline: "Moving fast, making changes daily, and releasing often",
```

Comment:
Not convinced by this - 'making changes daily' is in 'implementation details' territory, and overall this feels more like a description of project management than the language itself

Reply:
I'll just remove it entirely and switch the flag back for now.

Reply:
I think we could add this as a one more section similar to "collaboration, research and usage", or maybe just a bullet point in that section. Something like

> ## Active Development
> HHVM and the Hack language are in active development. We are moving fast, making changes daily and releasing often. If you notice a regression in the typechecker or the runtime, please [open issues](https://github.com/facebook/hhvm/issues/new) when you find them.

Reply:
Sure, I can add it as an introduction <p>